run-tests

```diff
@@ -1,16 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
No, thanks for pointing this out! I had a bunch of files that were no longer used in `sparkworker` and this one got mixed with them, reverted that commit.